src/app/services/mqtt/mqtt.service.ts

```diff
@@ -174,29 +173,29 @@ export class MQService implements TransportService {
   public debug(...args: any[]) {
 
     // Push arguments to this function into console.log
-    if (window.console && console.log && console.log.apply) {
+    if (console && console.log && console.log.apply) {
```

Comment:
I removed the `window` reference as it seems redundant. I tested the changes and the result is the same.